src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -65,6 +55,14 @@ export class ConsoleMenu extends Component<Props, State> {
 
   copyAsCurl() {
     this.copyText(this.state.curlCode);
+    const { addNotification } = this.props;
+    if (addNotification) {
+      addNotification({
+        title: i18n.translate('console.consoleMenu.copyAsCurlMessage', {
+          defaultMessage: 'Copied as cURL!',
```

Comment:
I think the original message "Request copied as cURL" was a little clearer/explicit. Also I think our writing style guide tends to discourage exclamation points. Though personally I like them!